lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -16,34 +16,12 @@
  */
 class WP_REST_Font_Faces_Controller extends WP_REST_Posts_Controller {
 	/**
-	 * The base of the parent Font Family controller's route.
+	 * Whether the controller supports batching.
 	 *
 	 * @since 6.5.0
-	 * @var string
+	 * @var false
 	 */
-	private $parent_base;
-
-	/**
-	 * Parent font family post type
-	 *
-	 * @since 6.5.0
-	 * @var string
-	 */
-	private $parent_post_type;
-
-	public function __construct() {
-		$post_type       = 'wp_font_face';
-		$this->post_type = $post_type;
-
-		$post_type_obj   = get_post_type_object( $post_type );
-		$this->rest_base = $post_type_obj->rest_base;
-
-		$parent_post_type       = 'wp_font_family';
-		$this->parent_post_type = $parent_post_type;
-		$parent_post_type_obj   = get_post_type_object( $parent_post_type );
-		$this->parent_base      = $parent_post_type_obj->rest_base;
-		$this->namespace        = $parent_post_type_obj->rest_namespace;
-	}
+	protected $allow_batch = false;
```

Comment:
Batch support is on by default. Here I'm turning it off for font families, because the batch endpoint doesn't really support file uploads.